tst/EnergyPlus/unit/MixedAir.unit.cc

```diff
@@ -0,0 +1,150 @@
+// EnergyPlus::MixedAir Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <MixedAir.hh>
+#include <OutAirNodeManager.hh>
+#include <InputProcessor.hh>
+#include <DataHeatBalance.hh>
+#include <DataIPShortCuts.hh>
+#include <DataGlobals.hh>
+#include <DataStringGlobals.hh>
+#include <ObjexxFCL/gio.hh>
+#include <DataLoopNode.hh>
+
+using namespace EnergyPlus;
+using namespace EnergyPlus::MixedAir;
+using namespace EnergyPlus::OutAirNodeManager;
+using namespace EnergyPlus::InputProcessor;
+using namespace EnergyPlus::DataIPShortCuts;
+using namespace EnergyPlus::DataGlobals;
+using namespace EnergyPlus::DataLoopNode;
+using namespace ObjexxFCL;
+
+TEST( ProcessOAControllerTest, Test1 )
+{
+// Test input processing of portion of Controller:OutdoorAir object
+	//  Controller:OutdoorAir,
+	//    OA Controller 1,         !- Name
+	//    Relief Air Outlet Node 1,!- Relief Air Outlet Node Name
+	//    VAV Sys 1 Inlet Node,    !- Return Air Node Name
+	//    Mixed Air Node 1,        !- Mixed Air Node Name
+	//    Outside Air Inlet Node 1,!- Actuator Node Name
+	//    autosize,                !- Minimum Outdoor Air Flow Rate {m3/s}
+	//    autosize,                !- Maximum Outdoor Air Flow Rate {m3/s}
+
+	static bool ErrorsFound( false ); // If errors detected in input
+	static int ControllerNum( 0 ); // Controller number
```

Comment:
@mjwitte Looking around some it looks like static is getting used a lot elsewhere, too. So maybe the best thing is to get rid of the static modifier here and leave it alone elsewhere. It might be a bigger task to look at all of the uses of static than we want to get involved in right now. How does that sound to you?